apps/zk-explorer/components/ItemGrid.tsx

```diff
@@ -0,0 +1,60 @@
+import { PropsOf } from '@webb-tools/webb-ui-components/types';
+import { FC } from 'react';
+import { ItemType, PageUrl } from '../utils/utils';
+import { ProjectItem } from './ProjectCard/types';
+import { ProjectCard } from './ProjectCard/ProjectCard';
+import { CircuitItem } from './CircuitCard/types';
+import { CircuitCard } from './CircuitCard/CircuitCard';
+import { Search } from '@webb-tools/icons';
+import { Button, Typography } from '@webb-tools/webb-ui-components';
+
+export type CardGridProps = PropsOf<'div'> & {
+  selectedItemType: ItemType;
+  projects: ProjectItem[];
+  circuits: CircuitItem[];
+};
+
+export const ItemGrid: FC<CardGridProps> = (props) => {
+  const items =
```

Comment:
add `useMemo` on both **items** and **wereResultsFound**

Reply:
For these two, I think it would be best to leave them without memoization, since they are trivial operations (constant time). Although they do get recomputed at every re-render, they are constant time operations so they should have the same performance impact as if I were to `useMemo` with them.

What do you think?

Reply:
Yes we can keep it this way for now and make new updates when integrating with back-end.